ethminer/MinerAux.h

```diff
@@ -315,7 +315,29 @@ class MinerCLI
 		}
 		else if ((arg == "--work-timeout") && i + 1 < argc)
 		{
-			m_worktimeout = atoi(argv[++i]);
+			try
+			{
+				m_worktimeout = atoi(argv[++i]);
```

Comment:
`atoi` does not throw any exception. only `stoi` does.

Reply:
Right. Too quick cut and paste

Better have stoi and handle error instead of setting a value to zero for invalid input.